src/components/calculator.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+import Btn from './buttons';
+
```

Comment:
- [ ] `calculator.js` file cannot be referenced in `App.js` because of mismatch in file name and import directory reference. Let's correct this by renaming the `calculator.js` file to `Calculator.js` to enable your app work correctly.